src/uShip.Logging.Tests/App.config

```
@@ -28,6 +28,7 @@
       <add field="creditCard" />
     </urlFormEncodedReplacements>
     <regexReplacements>
+      <add field="[p|P]assword=.*" />
```

It looks like the casing of these pattern is automatically ignored
https://github.com/uShip/uShip.Logging/blob/aec6f6882b64a4562f046db719dd82981cca9836/src/uShip.Logging/LogBuilders/LogDataSanitizeExtensions.cs#L59
and the `=.*` is the essential difference here.  So I think this could be simplified to
```suggestion
      <add field="password=.*" />
```